root/entity/details.tt

```diff
@@ -1,46 +0,0 @@
-[% WRAPPER "$entity_type/layout.tt" title=l('Details') page='details' %]
-    [%~ entity_type_for_url = entity_properties.url || entity_type ~%]
-    <h2>[%~ l('Details') ~%]</h2>
-    <table class="details">
-        <tr>
-            <th>[%~ l('Name:') ~%]</th>
-            <td>[%~ link_entity(entity) ~%]</td>
-        </tr>
-        <tr>
-            <th>[%~ l('{mbid|<abbr title="MusicBrainz Identifier">MBID</abbr>}:',
-                    { mbid => doc_link("MusicBrainz_Identifier") }) ~%]
-            </th>
-            <td><code>[%~ entity.gid ~%]</code></td>
-        </tr>
-        <tr>
-            <th>[%~ l('Last updated:') ~%]</th>
-            <td>[%~ UserDate.format_pg_string(entity.last_updated) || l('(unknown)') ~%]</td>
-        </tr>
-        <tr>
-            <th>[%~ l('Permanent link:') ~%]</th>
-            <td>
-                [%~ USE Canonicalize ~%]
```

Comment:
TATBTLUO lib/MusicBrainz/Server/Plugin/Canonicalize.pm so we can remove that file.